packages/skill-api/src/core/services/stripe-checkout.ts

```diff
@@ -24,6 +26,7 @@ export async function stripeCheckout({
 }): Promise<OutgoingResponse> {
   try {
     const {req} = params
+    const token = await getToken({req: req as unknown as NextApiRequest})
```

Comment:
we are passing in a userId, but we can just grab a token and it's a lot more consistent so we are attaching new purchases to the existing user's customer in stripe